lua/nvim-treesitter/parsers.lua

```diff
@@ -1443,6 +1443,17 @@ list.ini = {
   experimental = true,
 }
 
+list.poe_filter = {
+  install_info = {
+    url = "https://github.com/ObserverOfTime/tree-sitter-poe-filter",
+    files = { "src/parser.c" },
+  },
+  maintainers = { "@ObserverOfTime" },
+  experimental = true,
+  filetype = "poefilter",
+  readme_name = "PoE item filter",
```

Comment:
maybe also add the link you gave us :smile:

Reply:
As a comment?

Reply:
so that it is visible in the README. So maybe a markdown link or just the link in parens.

Reply:
I think it's enough to spell it out ("Path of Exile").